pkgs/applications/video/gnomecast/default.nix

```diff
@@ -16,6 +16,10 @@ buildPythonApplication rec {
     gtk3 gobject-introspection
   ];
 
+  # NOTE: gobject-introspection does not run with strictDeps
```

Comment:
In this case this is actually about gdk-pixbuf setup hook having the same issue. Could you adjust the comment to reflect that?